docs/api/download-item.md

```diff
@@ -184,6 +184,16 @@ Returns `string` - ETag header value.
 Returns `Double` - Number of seconds since the UNIX epoch when the download was
 started.
 
+[comment]: # (OpenFin Instance Methods)
+
+#### `downloadItem.getLastReason()`
```

Comment:
We should consider what we name this: `getLastReason()` isn't very indicative of this being the most recent interrupt reason. Perhaps something more like `getLastInterruptReason()`?

Reply:
getLastInterruptReason seems like a more reasonable name :)